components/menu/__tests__/type.test.tsx

```diff
@@ -37,4 +39,61 @@ describe('Menu.typescript', () => {
 
     expect(menu).toBeTruthy();
   });
+
+  it('Menu.items Customizable attributes', () => {
+    interface CustomItem extends MenuProps {
+      items: ItemType<{ 'data-x': number }>[];
```

Comment:
```suggestion
      items: MenuProps['items'][number]<{ 'data-x': number }>[];
```

是不是可以这样，用户不应该去 useItems 下找 type。

Reply:
> items: MenuProps['items']<{ 'data-x': number }>;

这种写法ts好像不支持吧？

Reply:
我意思是 ItemType 不在顶级入口中，不应该鼓励开发者去一个子目录找 type，有可能在重构中被改坏。

Reply:
ping